src/index.js

```diff
@@ -70,17 +70,18 @@ export default async function loader(content, sourceMap, meta) {
   );
 
   if (useSourceMap) {
-    processOptions.map = { inline: false, annotation: false };
+    if (
+      typeof processOptions.map === 'undefined' ||
+      typeof processOptions.map === 'boolean'
+    ) {
+      processOptions.map = { inline: false, annotation: false };
+    }
 
-    if (sourceMap) {
+    if (sourceMap && !processOptions.map.inline) {
       processOptions.map.prev = normalizeSourceMap(sourceMap, this.context);
+    } else {
+      processOptions.map.prev = sourceMap;
```

Comment:
When we set `sourceMap: true` we should always normalize source map, otherwise you will get unstable `contenthash` on different platforms due absolute paths in sources from other loaders

Reply:
Updated